test/common/testlib.py

```diff
@@ -817,7 +821,7 @@ def check_journal_messages(self, machine=None):
                     found = True
                     break
             if not found:
-                print("Unexpected journal message '%s'" % m)
+                print(m)
```

Comment:
This might be a bit confusing especially for newcomers; could we just keep the explanation? A core dump is sort of a message, too?